static/styles/zulip.scss

```diff
@@ -557,49 +562,39 @@ td.pointer {
 .message_edit_notice {
     font-size: 10px;
     opacity: 0.5;
-    line-height: 0px;
-    text-align: right;
-    width: 45px;
-    position: absolute;
-    left: 0px;
-    top: 16px;
+    line-height: 23px;
```

Comment:
Where doe this `23px` value come from?

Reply:
That's because I needed to bring the edited label vertically middle to with reference to the the message content (as this was the case previously). A normal messagebox paragraph has top margin of 3px and a line-height of 17px (where the font-size is 14px). Hence the text had to be 2px above the bottom line of messagebox content since the edited text is of the size 10px (`(14 - 10) / 2`). Height left to increase is now: `3px` (of the top margin) + `1.5px` (half of the difference between line-height and font-size) and `12px` (14 - 2) minus the `10px` of font size. That's `6.5px`, so increase in line height = `6.5 * 2 = 13px`. Hence new line-height = `10 + 13 = 23`px.